### ISSUE TITLE
Exception when try to generate test or go to test

### ISSUE THREAD

**Issue by marbel82** [mattwhitfield/Unitverse#88]
Environment: Visual Studio 2022  (Version 17.2.0 Preview 1.0)
Unitverse: 0.113.0.0

When I Click "Generate test for method ...", I get a MessageBox with the content:
```
Unitverse

Exception raised
System.NotImplementedException: Nie zaimplementowano. (Exception from HRESULT: 0x80004001 (E_NOTIMPL))

   at EnvDTE.Project.get_FileName()
   at Unitverse.Helper.VsProjectHelper.<FindProjects>d__5.MoveNext() in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\VsProjectHelper.cs:line 60
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Unitverse.Helper.ProjectMappingFactory.CreateMappingFor(Project sourceProject, IUnitTestGeneratorOptions baseOptions, Boolean interactive, Boolean skipManuallySelectedTargetResolution) in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\ProjectMappingFactory.cs:line 26
   at Unitverse.Commands.GenerateTestForSymbolCommand.<>c__DisplayClass13_0.<Execute>b__0() in D:\a\Unitverse\Unitverse\src\Unitverse\Commands\GenerateTestForSymbolCommand.cs:line 205
   at Unitverse.Helper.Attempt.Action(Action action, IUnitTestGeneratorPackage package) in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\Attempt.cs:line 18
```

When I Click "Go to tests", I get a MessageBox with the content:
```
Unitverse

Exception raised
System.NotImplementedException: Nie zaimplementowano. (Exception from HRESULT: 0x80004001 (E_NOTIMPL))

   at EnvDTE.Project.get_FileName()
   at Unitverse.Helper.VsProjectHelper.<FindProjects>d__5.MoveNext() in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\VsProjectHelper.cs:line 60
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at Unitverse.Helper.ProjectMappingFactory.CreateMappingFor(Project sourceProject, IUnitTestGeneratorOptions baseOptions, Boolean interactive, Boolean skipManuallySelectedTargetResolution) in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\ProjectMappingFactory.cs:line 26
   at Unitverse.Helper.GoToTestsHelper.FindTestsFor(ProjectItemModel source, IUnitTestGeneratorPackage package) in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\GoToTestsHelper.cs:line 14
   at Unitverse.Commands.GoToUnitTestsForSymbolCommand.<Execute>b__8_0() in D:\a\Unitverse\Unitverse\src\Unitverse\Commands\GoToUnitTestsForSymbolCommand.cs:line 107
   at Unitverse.Helper.Attempt.Action(Action action, IUnitTestGeneratorPackage package) in D:\a\Unitverse\Unitverse\src\Unitverse\Helper\Attempt.cs:line 18

```

**Comment by mattwhitfield** [comment]
Can you let me know what your project hierarchy looks like and what project types you have in there? It will basically be that you have a 'project' in the hierarchy that doesn't have an associated file name.

I'll look to push a generic fix for that tonight, though.

**Comment by marbel82** [comment]
Here is my solution explorer looks like:
![image](https://user-images.githubusercontent.com/31281571/158610318-759bb6b6-3596-4a4a-b845-7f0fad4927c6.png)

I don't know when "External Sources" appear. I haven't had that before.

**Comment by mattwhitfield** [comment]
I wonder if it's the unloaded projects... But either way the fix I am putting in should cover it 👍

**Comment by mattwhitfield** [comment]
Resolved in 0.119 🎉
